packages/legend-extension-dsl-data-space-studio/src/components/DataSpaceEditor.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import { useEditorStore } from '@finos/legend-application-studio';
+import { PanelFormTextField, PanelFormSection } from '@finos/legend-art';
+import { DataSpaceEditorState } from '../stores/DataSpaceEditorState.js';
+
+export const DataSpaceEditor = observer(() => {
+  const editorStore = useEditorStore();
+
+  const formEditorState =
+    editorStore.tabManagerState.getCurrentEditorState(DataSpaceEditorState);
+
+  const formElement = formEditorState.dataSpace;
+
+  const handleTitleChange = (value: string | undefined) => {
+    formElement.title = value ?? '';
+  };
+
+  const handleDescriptionChange = (value: string | undefined) => {
+    formElement.description = value ?? '';
+  };
+
+  return (
+    <div className="form-text-editor panel text-element-editor">
```

Comment:
why has there been no css work done in this PR ? 
you are using css names that don't exists or are you using ones from the text-element-editor.
Please set up a css for dataspace if one doesnt exists and properly align the css to the rest of the form. I have added comments on the video recording. the css does not look like the rest of the form.

Reply:
stylings added